documentation/src/main/resources/pages/ditto/release_notes_200.md

```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
+Ditto 2.0.0. If you need to migrate "things" from API 1 to API version 2, please have a look at the  
+[migration notes](#migrate-api-1-things-to-api-2).
+
+#### Removal of deprecated code + binary incompatible changes to Java APIs
+
+In order to not break binary compatibility in Ditto 1.x, existing APIs were marked as `@Deprecated` with a comment 
+pointing to an alternative implementation to use instead. Now, these deprecated APIs are removed from Ditto's codebase.
+
+Some changes to the codebase which could not be done in Ditto 1.x withour breaking binary compatibility were also done.
+
+#### [Removed content-type header mapping for connection targets](https://github.com/eclipse/ditto/pull/934)
+
+Removed the default header mapping of `content-type` for new connection targets. The header mapping led to irritating
+results, when payload mapping and header mapping disagreed on the actual `content-type`. Existing connections will still
+keep the "old" default and map the `content-type` header.
+
+If you need to keep the old behavior, please have a look at the  
+[migration notes](#content-type-header-mapping-in-connection-targets).
+
+#### OpenID Connect configuration change
+
+For supporting [Configurable OpenID Connect / OAuth2.0 claim extraction](https://github.com/eclipse/ditto/issues/512), 
+the configuration format was changed, please have a look at the  
+[migration notes](#openid-connect-configuration-for-gateway).
+
+#### Removal of header `x-ditto-dummy-auth`
+
+The HTTP header / query param `x-ditto-dummy-auth` which was already an alias for the 
+[pre-authentication provider](installation-operating.html#pre-authentication) header `x-ditto-pre-authenticated` has
+been removed from Ditto 2.0.  
+Please use the header `x-ditto-pre-authenticated` instead.
+
+#### Removed default source header mapping for MQTT connections
+
+The default source header mapping of MQTT connections was removed. The headers `mqtt.topic`, `mqtt.qos`
+and `mqtt.retain` now must explicitly be added to the source header mapping if they are required for further processing.
+
+
+#### Restructuring of Ditto's Maven modules
+
+Ditto's modules were adjusted to be structured in a more functional way. In Ditto 1.x the modules were structured
+in a more technical way.
+
+This table shows the old modules and in which module the old ones can be found in Ditto 2.0.0:
+
+| Ditto 1.x module                             | Ditto 2.x module          |
+| ---                                          | ---                       |
+| `ditto-model`                                | `-` (was pom only)        |
+| `- ditto-model-base`                         | `ditto-base/ditto-base-model` |
+| `- ditto-model-cleanup`                      | `-` (was internal API) |
+| `- ditto-model-connectivity`                 | `ditto-connections/ditto-connections-model` |
+| `- ditto-model-devops`                       | `ditto-devops/ditto-devops-model` |
```

Comment:
ditto-model-devops was merged into ditto-base/ditto-base-api (org.eclipse.ditto.base.api.devops) and is considered to be internal API